src/terraform/modules/aws/public-service/alb.tf

```diff
@@ -51,16 +49,3 @@ resource "aws_alb_listener" "external" {
     type             = "forward"
   }
 }
-
-resource "aws_alb_listener" "internal" {
-  count = "${var.internal ? 1 : 0}"
```

Comment:
Even though there's some repetition because of the `depends_on` I don't hate it because differences in this abl_listener makes this more easily configurable in the future.

👍